Core/HLE/sceIo.cpp

```diff
@@ -2067,11 +2069,15 @@ static u32 sceIoDevctl(const char *name, int cmd, u32 argAddr, int argLen, u32 o
 				Memory::Write_Float(scale, outPtr);
 			}
 			return 0;
-		case EMULATOR_DEVCTL__GET_LTRIGGER:
-			//To-do
+		case EMULATOR_DEVCTL__GET_AXIS:
+			if (Memory::IsValidAddress(outPtr)) {
+				Memory::Write_Float(HLEPlugins::PluginDataAxis[argAddr], outPtr);
```

Comment:
argAddr (which is being used as an index) needs range checking here, otherwise the plugin can easily crash the emulator here. Same below.